backends/dpdk/dpdkMetadata.h

```diff
@@ -0,0 +1,65 @@
+/*
+Copyright 2022 Intel Corp.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+#ifndef BACKENDS_DPDK_DPDKMETADATA_H_
+#define BACKENDS_DPDK_DPDKMETADATA_H_
+
+#include "frontends/common/resolveReferences/referenceMap.h"
+#include "ir/ir.h"
+
+namespace DPDK {
+
+class AddNewMetadataFields : public Transform {
```

Comment:
Do you want to add a comment somewhere describing what these passes collectively do (with a small example)?